hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieBaseRelation.scala

```diff
@@ -155,12 +158,13 @@ abstract class HoodieBaseRelation(val sqlContext: SQLContext,
       }
     }
 
+    val avroNameAndSpace = AvroConversionUtils.getAvroRecordNameAndNamespace(tableName)
```

Comment:
nit: we can move this inside the map and also make it `val (name, namespace) = getAvroRecordNameAndNamespace(...)`

Reply:
fixed